ledger/sandbox/src/main/resources/db/migration/h2database/V34__sequential_row_id.sql

```diff
@@ -0,0 +1,23 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V34: Add a sequential row_id to participant_events
+--
+---------------------------------------------------------------------------------------------------
+
+-- 1. add the column
+alter table participant_events add column row_id bigint auto_increment;
```

Comment:
I would suggest naming this so that's it's clear it's supposed to be sequential, in particular with regards to the history. What about `event_sequential_id`?

Reply:
It is `participant_events` table I don't think we need to say `event_` one more time, `row_id` has `auto_increment` type, which is self descriptive. However if you insist on calling it `event_sequential_id` I guess it would make it consistent with `event_id` and `event_offset`. Up to you.

Reply:
I understand it's part of the type and that it can feel as if it's repeating unnecessary information, but I would prefer if its purpose could be clearer from the name, without having to go to the definition. `row_id` feels a bit too generic. I would prefer having a more telling name.

Reply:
all right, I will rename the column to `event_sequential_id`